javascript/package-lock.json

```diff
@@ -1,8 +1,1265 @@
 {
```

Comment:
I don't think you'll want to commit this file

Reply:
Reason for the change is because `axios` has been added to `auth-sample-implementations` so I think this file needs committing for the cohesion of dependencies

Reply:
did you run `npm install` or `yarn install`? It normally gets added when you run `npm`. This file doesn't exist in our repos because we have `yarn.lock` so you can normally just delete it as long as it's updated the `yarn.lock` - unless there's another reason for it?

Reply:
`auth-sample-implementations` uses the npm package manager

Reply:
Yep, we'd expect `yarn.lock` over `package-lock.json` - so would expect the former to be checked in here as it doesn't currently exist

Reply:
Though, the Dockerfile and CI stuff uses `npm`...

Reply:
Ah yes just seen this repo uses npm.. nice and consistent 😄 but does seem to have large amount of changes in it so not sure if there's an error with it as can see in master the `package-lock.json` file exists?

Reply:
Big assumption here but it may be that we use npm in this public repo due to the ubiquity of it

Reply:
Ah yes just seen this repo uses npm.. but does seem to have large amount of changes in it so not sure if there's an error with it as can see in master the package-lock.json file exists?

Reply:
It could be that npm update hasn't been run for a while and that `axios` is a big old package. Most of the dependencies in there are http-focused